frontend/src/app/shared/shared.module.ts

```diff
@@ -23,6 +24,8 @@ import { RouterModule } from '@angular/router';
         }),
         NgbModule,
         RouterModule,
+        FormsModule,
```

Comment:
We're not using template-driven forms, so we don't need the `FormsModule`. The project compiles fine if I remove it from `import` and `export` (also in the `shared-testing.module.ts`).